### PR TITLE
Port makunbound

### PR DESCRIPTION
**Pull request by shanavas786**

### CODE REVIEW

File: rust_src/src/symbols.rs [hunk 1]
```diff
@@ -170,3 +175,19 @@ fn indirect_variable_lisp(object: LispObject) -> LispObject {
         object
     }
 }
+
+
+/// Make SYMBOL's value be void.
+/// Return SYMBOL.
+#[lisp_fn]
+fn makunbound(symbol: LispObject) -> LispObject {
+    if let Some(sym) = symbol.as_symbol() {
```
Comment (shaleh) [inline rust_src/src/symbols.rs]: Would this not be better as:

    let sym = symbol.as_symbol_or_error();
    if sym.is_contant() {
        xsignal!(Qsetting_constant, symbol);
    }
    unsafe {
        Fset(symbol.to_raw(), Qunbound);
    }
    symbol

File: rust_src/src/symbols.rs [hunk 2]
```diff
@@ -170,3 +175,19 @@ fn indirect_variable_lisp(object: LispObject) -> LispObject {
         object
     }
 }
+
```
Comment (shaleh) [inline rust_src/src/symbols.rs]: Double empty lines

File: rust_src/src/symbols.rs (hunk 1 above)
Comment (shanavas786) [reply rust_src/src/symbols.rs]: thanks, updated

**Comment by brotzeit** [comment]
Can you merge this as a squash commit the next time @shanavas786.

**Comment by shanavas786** [comment]
Sure, I usually prefer squash merge. I created merge commit here because most of the PRs where merged that way.

**Comment by brotzeit** [comment]
I know, the topic came up in #393.

**Pull request merged**